fuzzers/fuzzbench/src/bin/libafl_cc.rs

```diff
@@ -16,6 +16,10 @@ pub fn main() {
         dir.pop();
 
         let mut cc = ClangWrapper::new();
+
+        #[cfg(target_os = "linux")]
+        cc.add_pass(LLVMPasses::AutoTokens);
```

Comment:
We probably want to add useful passed automatically in the future, or at least have an interface to add them in aflpp_cc, what do you think @andreafioraldi